src/rgw/driver/rados/rgw_data_sync.cc

```diff
@@ -300,12 +301,16 @@ struct read_remote_data_log_response {
   string marker;
   bool truncated;
   vector<rgw_data_change_log_entry> entries;
+  real_time last_update;
 
   read_remote_data_log_response() : truncated(false) {}
 
   void decode_json(JSONObj *obj) {
     JSONDecoder::decode_json("marker", marker, obj);
     JSONDecoder::decode_json("truncated", truncated, obj);
+    utime_t lu;
```

Comment:
Can we use `real_time` here instead? I think it should be JSON decodable.

Reply:
updated, thanks